roles/chart_verifier/tasks/tests.yml

```diff
@@ -90,29 +90,25 @@
         - chart_stat is defined
         - chart_stat.stat.exists
 
-    - name: "Check values_file stat"
+    - name: "Check if chart.values_file exist"
       stat:
```

Comment:
Hi Beto,
What if the values_file list include a web URL? This task will return false even if the URL is valid.

Reply:
This block is to check local files